pkg/test/framework/runtime/components/galley/native.go

```diff
@@ -37,10 +37,10 @@ import (
 )
 
 var (
-	_ components.Galley = &nativeComponent{}
-	_ api.Component     = &nativeComponent{}
-	_ io.Closer         = &nativeComponent{}
-	_ api.Resettable    = &nativeComponent{}
+	_ components.Galley = &nativeGalley{}
```

Comment:
nit: not sure this is a positive change. It's more idiomatic in Go to leave out the package name from type names to avoid stutter.  Since we're in a package called "galley", having "Galley" in the name is redundant. It's more specifically a `galley.xxxComponent`. Although `components` appears in the package hierarchy, Go only ever deals with the most specific package (i.e. you can't do `test.framework.runtime.galley.xxx` like you would in Java).

Reply:
hmm, I guess I have to learn a new idiom when searching code for types. I'm used to using an IDEs mechanism to find types by names but I guess I need to qualify them with their package to get useful resolution

Will revert

Reply:
Done